app/controllers/simulations_controller.rb

```diff
@@ -334,8 +336,17 @@ def mark_as_complete
               unless sm_record.computational_resources.blank?
                 @simulation_run.computational_resources = sm_record.computational_resources
               end
+
+              sm_record.simulations_left -= 1 if sm_record.simulations_left
+              sm_record.finished_simulations ||= 0
+              sm_record.finished_simulations += 1
+              sm_record.save
             end
 
+            if params.include? :execution_statistics
```

Comment:
OK, rozumiem. To teraz napiszcie, bo będzie akcja z dokumentowaniem całości i ten fragment się przyda.